ibis/backends/conftest.py

```diff
@@ -258,10 +445,556 @@ def pytest_runtest_call(item):
             )
 
 
+def load_postgres(data_dir: Path, script_dir: Path, **kwargs) -> None:
+    """Load testdata into the postgres backend.
+
+    Parameters
+    ----------
+    data_dir : Path
+        Location of testdata
+    script_dir : Path
+        Location of scripts defining schemas
+    """
+    PG_USER = (
+        os.environ.get(
+            'IBIS_TEST_POSTGRES_USER', os.environ.get('PGUSER', 'postgres')
+        )
+        if not kwargs.get("username", None)
```

Comment:
These `kwarg` checks were added to support users using non-default values for `ci/datamgr.py`. It does add a lot of boiler plate. 

Do we want to continue supporting those `kwargs`? Boiler plate could also be decreased by me assuming that the `kwargs` for all of the values will be set, if there are `kwargs` being set. Then, I could just do an `if kwargs` check, and set variables appropriately.